packages/arb-token-bridge-ui/src/components/TransferPanel/TransferDisabledDialog.tsx

```diff
@@ -0,0 +1,55 @@
+import { create } from 'zustand'
+
+import { useActions, useAppState } from '../../state'
+import { Dialog } from '../common/Dialog'
+
+type TransferDisabledDialogStore = {
+  isOpen: boolean
+  openDialog: () => void
+  closeDialog: () => void
+}
+
+export const useTransferDisabledDialogStore =
+  create<TransferDisabledDialogStore>(set => ({
+    isOpen: false,
+    openDialog: () => set({ isOpen: true }),
+    closeDialog: () => set({ isOpen: false })
+  }))
+
+export function TransferDisabledDialog() {
+  const { app } = useAppState()
+  const { selectedToken } = app
+  const {
+    app: { setSelectedToken }
+  } = useActions()
+  const {
+    isOpen: isOpenTransferDisabledDialog,
+    closeDialog: closeTransferDisabledDialog
+  } = useTransferDisabledDialogStore()
+  const unsupportedToken = selectedToken?.symbol ?? 'this token'
```

Comment:
This should never happen tho? If the selected token is undefined it means it's the native token

Reply:
yea i think so, i probably hit a weird case when i was working on it but it no longer happens after it's done